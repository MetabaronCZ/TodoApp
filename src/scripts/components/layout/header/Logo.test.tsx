import React from 'react';

import { render } from '@testing-library/react';
import { describe, expect, it } from 'vitest';

import { withMockedProviders } from 'test/component';
import { Logo } from 'components/layout/header/Logo';

describe('components/layout/Logo', () => {
  it('should render correctly', () => {
    const { container } = render(withMockedProviders(<Logo />));
    expect(container).toMatchSnapshot();
    expect(container.querySelector('[href="/"]')).toBeInTheDocument();
  });
});
