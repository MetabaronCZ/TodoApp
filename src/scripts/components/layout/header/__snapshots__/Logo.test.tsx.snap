// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`components/layout/Logo > should render correctly 1`] = `
.c0 {
  font-family: Open Sans,Arial,sans-serif;
  color: #222;
  font-size: 14px;
  line-height: 20px;
  display: block;
  padding: 0 10px;
  font-size: 20px;
  line-height: 40px;
  font-weight: bold;
  -webkit-letter-spacing: 2px;
  -moz-letter-spacing: 2px;
  -ms-letter-spacing: 2px;
  letter-spacing: 2px;
  -webkit-text-decoration: none;
  text-decoration: none;
}

.c0:focus {
  outline: 1px dotted #222;
  outline-offset: -1px;
}

<div>
  <a
    class="c0"
    href="/"
  >
    â˜‘ 
    TODOLIST
  </a>
</div>
`;
